frontends/p4/fromv1.0/converters.h

```diff
@@ -89,7 +89,8 @@ class ExternConverter {
     virtual const IR::Type_Extern *convertExternType(ProgramStructure *,
                 const IR::Type_Extern *, cstring);
     virtual const IR::Declaration_Instance *convertExternInstance(ProgramStructure *,
-                const IR::Declaration_Instance *, cstring, IR::IndexedVector<IR::Declaration> *);
+                const IR::Declaration_Instance *, cstring, IR::IndexedVector<IR::Declaration> *,
+                IR::IndexedVector<IR::Node> *);
```

Comment:
should have comments or names here to make clear which argument is the local scape and which is the global scope.